README.md

```diff
@@ -43,6 +43,12 @@ As a system administrator you can also install it through the TAO Extension Mana
 - Settings (the gears on the right hand side of the menu) -> Extension manager
 - Select _taoPublishing_ on the right hand side, check the box and hit _install_
 
+## Remote-publishing from test
+To enable set feature flag environment variable:
+```
+FEATURE_FLAG_REMOTE_PUBLISHING_FROM_TEST = true
```

Comment:
Why a feature flag here?

Reply:
The extension that is affected by the feature flag is tao-test, not this one. So I would still keep it there. For the matter of fact, what we want to do it to enable or not an extra property on the test, so I would even change the feature flag name to something else.

Even if @pnal will use the `AbstractFeatureFlagFormPropertyMapper` to map the test fields based on feature flag, I would still prefer to keep the feature flag in the place it is affecting.

Reply:
ok ok. For now, we are going to postpone it and schedule a meeting with Pedro to decide what to do.